wallet/src/wallet/tests.rs

```diff
@@ -2275,6 +2304,961 @@ fn issue_and_transfer_tokens(#[case] seed: Seed) {
     }
 }
 
+#[rstest]
+#[trace]
+#[case(Seed::from_entropy())]
+fn check_tokens_v0_are_ignored(#[case] seed: Seed) {
+    let mut rng = make_seedable_rng(seed);
+    let chain_config = Arc::new(create_mainnet());
```

Comment:
Is there a reason why we're using mainnet here? Just understand that this can change in the future.

Reply:
no particular reason, I will change it